app/models/order.rb

```diff
@@ -0,0 +1,20 @@
+class Order < ApplicationRecord
+  enum status: [:pending, :shipped, :delivered, :canceled]
```

Comment:
```suggestion
  enum status: {pending: 0, shipped: 1, delivered: 2, canceled: 3}
```
dùng dạng hash thay vì array bé nha